### PR TITLE
Parameters Annotations Code Added

### PR DESCRIPTION
**Pull request by budheeswar**
This PR is about,
I added  Parameter Annotation Usage by taking a Login Credential Example, this code explains getting Parameter value from XML file to Java Code 

@Buddeeswar please validate the code and Merge it to master branch ,

### CODE REVIEW

File: src/test/resources/ParamAnotTestNG.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	<parameter name="global" value="available in entire suite"></parameter>
+	<test thread-count="5" name="TestName1">
+		<parameter name="url"
+			value="https://practicetestautomation.com/practice-test-login/" />
+		<parameter name="name" value="student"></parameter>
+		<parameter name="password" value="Password123"></parameter>
+		<classes>
+			<class name="com.rb.second.ParamAnotationLoginEx">
+				<methods>
+					<include name="testLogin"></include>
```
Comment (Buddeeswar) [inline src/test/resources/ParamAnotTestNG.xml:13]: Intentionally commenting to see github action

**Review by Buddeeswar — COMMENTED** [review]
pending one comment

File: src/test/resources/ParamAnotTestNG.xml (hunk 1 above)
Comment (budheeswar) [reply src/test/resources/ParamAnotTestNG.xml:13]: ok, so now please approve

**Review by Buddeeswar — APPROVED** [review]
approved

**Pull request merged**